docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,885 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    T = np.random.rand(100000)
</pre><p>I increased the length of <code>T</code> to  better show the difference between naive version and DAMP algo. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>